lambda-functions/receiveNotificationRequest/src/main/java/com/paladincloud/SendNotification.java

```diff
@@ -0,0 +1,56 @@
+package com.paladincloud;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.sns.AmazonSNS;
+import com.amazonaws.services.sns.AmazonSNSClientBuilder;
+import com.amazonaws.services.sns.model.PublishRequest;
+import com.amazonaws.services.sns.model.PublishResult;
+import com.google.gson.*;
+
+import java.util.List;
+import java.util.Map;
+
+
+public class SendNotification implements RequestHandler<Map<String,Object>, String> {
+    Gson gson = new GsonBuilder().setPrettyPrinting().create();
+
+    /**
+     *Below method places all incoming notification requests into topic which are consumed
+     * by subsequent lambda functions.
+     *
+     */
+    @Override
+    public String handleRequest(Map<String,Object> event, Context context)
+    {
+        LambdaLogger logger = context.getLogger();
+        String response = new String("hello paladin cloud");
```

Comment:
this should be an empty string, or a better meaningful placeholder

Reply:
will change it in the next PR.